src/lib/tvgPaint.h

```diff
@@ -50,6 +51,7 @@ namespace tvg
         CompositeMethod cmpMethod = CompositeMethod::None;
         PaintType type;
         uint8_t opacity = 255;
+        unique_ptr<Saver> saver = nullptr;
```

Comment:
I think we should not keep this saver since it's not common case. it can be used temporarily.